app/views/medicaid/intro_location/edit.html.erb

```diff
@@ -9,24 +9,30 @@
 
 <div class="form-card">
   <header class="form-card__header">
-    <div class="form-card__title">Before we get started, do you currently reside in Michigan?</div>
+    <div class="form-card__title">
+      Before we get started, do you currently reside in Michigan?
+    </div>
+
     <p class="text--help text--centered">
-      If not, we'll help point you in the right direction. However, this application for health coverage is specifically for current residents of Michigan.
+      If not, we'll help point you in the right direction. However, this
+      application for health coverage is specifically for current residents of
+      Michigan.
     </p>
   </header>
 
-  <div class="form-card__content">
-    <!-- Replace with partial -->
-    <footer class="form-card__buttons">
-      <a href="/pages/medicaid/intro_location_help" class="button button--nav button--full-width">
-        No
-      </a>
+  <%= form_for @step,
+    as: :step,
+    builder: MbFormBuilder,
+    url: current_path,
+    method: :put do |f| %>
 
-      <a href="/pages/medicaid/intro_name" class="button button--nav button--cta button--full-width">
-        Yes
-      </a>
-    </footer>
-  </div>
-</div>
+    <div class="form-card__content">
+      <footer class="form-card__buttons">
+        <%= f.mb_boolean_buttons :michigan_resident,
+          no_path: "/pages/medicaid/intro_location_help",
```

Comment:
the way we do this logic in the snap appl flow is that the next step is always `intro_location_help` but that controller has a `skip?` method that skips based on the previous answer - I think we should stick to that pattern for consistency, which will mean that both yes and no will have the same path and we don't need to worry about `redirect_path` stuff in here